src/app/pages/homepage/homepage.component.html

```diff
@@ -49,35 +49,31 @@
         <a class="button-ask">Ask Mentor</a>
       </div>
       <div class="news-block">
+        <h3>News</h3>
         <ul>
-          <li>
-            <i>19-03-1974</i>
-            <h3>Title</h3>
-            <p>Lorem ispum dolor sit amet
-              Lorem ispum dolor sit amet </p>
-            <a>More</a>
-          </li>
-          <li>
-            <i>19-03-1974</i>
-            <h3>Title</h3>
-            <p>Lorem ispum dolor sit amet
-              Lorem ispum dolor sit amet </p>
-            <a>More</a>
+          <li *ngFor="let news of messageService.newsBlock">
+            <i>{{ news.timestamp }}</i>
+            <p>{{ news.text }}</p>
           </li>
         </ul>
+        <a *ngIf="this.adminUserId" (click)="openInput()">New News</a>
```

Comment:
Instead of *ngIf="this.adminUserId" check *ngIf="this.appUser && (this.appUser.role === 'admin' || this.appUser.role === 'super')"